internal/participate/support-the-network/about-nodes.md

```diff
@@ -20,48 +20,28 @@ Nodes are responsible for the following:
 
 ## Attaching new transactions to the Tangle
 
-When nodes receive a new transaction, they attach it to the Tangle by adding it to their local database.
+When nodes receive a new transaction, they attach it to the Tangle by adding it to their local database. As a result, at any point in time, all nodes may have different transactions in their local databases. These transactions make up a node's view of the Tangle. To distribute the transactions across the rest of the network, nodes synchronize their local databases with their neighbors.
 
-As a result, at any point in time, all nodes may have different transactions in their local databases. These transactions make up a node's view of the Tangle.
+## Network synchronization
 
-To distribute the transactions across the rest of the network, nodes synchronize their local databases with their neighbors.
+Like any distributed system, nodes in an IOTA network synchronize their databases with other nodes (neighbors) to form a single source of truth. When one node, no matter where it is in the world, receives a transaction, it will try to gossip it to all its neighbors. This way, all nodes eventually see all transactions and store them in their local databases.
 
-## Synchronizing with the rest of the network
+To synchronize, nodes in IOTA networks use milestones. If a node has the history of transactions that a milestone references in his database, that milestone is solid. Therefore, nodes know if they are synchronized if the index of their latest solid milestone is the same as the index of the latest milestone that it has received. When a node is synchronized, it then has enough information to decide which transactions it considers confirmed.
 
-Like any distributed system, nodes in an IOTA network synchronize their databases with others called neighbors to form a single source of truth.
+## Transaction confirmation
 
-When one node, no matter where it is in the world, receives a transaction, it will try to gossip it to all its neighbors. This way, all nodes eventually see all transactions and store them in their local databases.
+All transactions remain in a pending state until the node is sure of its validity. However, even when a transaction is valid, nodes may not be able to make a decision like in the case of a double spend. When nodes detect double spends, they must decide which transaction to consider confirmed and which one to ignore. Nodes do this by using consensus rules that are built into their node software. See [here](/learn/about-iota/an-introduction-to-iota#consensus-in-the-tangle) for more information about the consensus rules in IOTA networks.
```

Comment:
```suggestion
All transactions remain in a pending state until the node is sure of its validity. However, even when a transaction is valid, nodes may not immediately be able to confirm a transaction due to conflicts, like in the case of a double spend. When nodes detect conflicting transactions, they must decide which transaction to consider confirmed. Nodes do this by using consensus rules that are built into their node software. See [here](/learn/about-iota/an-introduction-to-iota#consensus-in-the-tangle) for more information about the consensus rules in IOTA networks.
```

Just emphasizing that not being able to decide is temporary. Also removed the part about transactions being ignored, because with weak referencing this is no longer the case.